app/utils/__init__.py

```diff
@@ -211,6 +211,16 @@ def get_defconfig_full(
 
 
 def make_path(dir_path):
+    """Created path with all intermediate directiories
```

Comment:
`Create` would be a bit more consistent, using present tense.

Reply:
Also if this code is being added in an earlier commit in this PR then this commit could be squashed with it.

Reply:
corrected